prisma/schema.prisma

```diff
@@ -1,69 +1,63 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
 datasource db {
-  provider     = "postgres"
+  provider     = "postgresql"
   url          = env("DATABASE_URL")
+  directUrl    = env("DIRECT_URL")
   relationMode = "prisma"
 }
 
-generator client {
-  provider = "prisma-client-js"
-}
-
 model Customer {
-  id                   String  @id @default(cuid())
-  name                 String  @unique
-  githubUserId         String? @unique
-  stripeSubscriptionId String? @unique
-  stripeCustomerId     String? @unique
-
-  projects Project[]
-
-  usage          Int      @default(0)
-  totalUsage     Int      @default(0)
-  usageLimit     Int      @default(20)
-  usageUpdatedAt DateTime @default(now())
-
-  usageWarned Boolean  @default(false)
-  createdAt   DateTime @default(now())
+  id                   String    @id @default(cuid())
+  name                 String    @unique
+  githubUserId         String?   @unique
+  stripeSubscriptionId String?   @unique
+  stripeCustomerId     String?   @unique
+  usage                Int       @default(0)
+  totalUsage           Int       @default(0)
+  usageLimit           Int       @default(20)
+  usageUpdatedAt       DateTime  @default(now())
+  usageWarned          Boolean   @default(false)
+  createdAt            DateTime  @default(now())
+  projects             Project[]
 }
 
 model Project {
-  id   String @id @default(cuid())
-  name String
-
-  customerId String
-  customer   Customer @relation(fields: [customerId], references: [id], onDelete: Cascade)
-
-  usage Usage[]
-
+  id                 String        @id @default(cuid())
+  name               String
+  customerId         String
   createdAt          DateTime      @default(now())
+  customer           Customer      @relation(fields: [customerId], references: [id], onDelete: Cascade)
+  usage              Usage[]
   customInstructions Instruction[]
 
   @@unique([customerId, name])
   @@index([customerId])
 }
 
 model Instruction {
-  id                String  @id @default(cuid())
+  id                String   @id @default(cuid())
   projectId         String
-  content           String  @db.Text
```

Comment:
Is this the only substantial change? If so, happy to merge!

Reply:
There's also a difference in the provider and direct/db URL. All pretty minor but very important.